.vscode/launch.json

```diff
@@ -6,11 +6,11 @@
   "configurations": [
     {
       "name": "meshtastic BLE",
-      "type": "python",
```

Comment:
(comment at random line in file)

I know basically nothing about vscode so I'm just gonna trust that these settings make sense. If someone else who actually uses vscode wants to validate, feel free.

Reply:
yah - these are just vscode warned me that "python" is deprecated and the new recommended string is pydebug...